features/step_definitions/loc_id_routing_steps.rb

```diff
@@ -24,10 +24,12 @@
 end
 
 Given(/^I visit my subject's locid$/) do
+  Capybara.current_session.driver.header('Accept', 'text/html')
```

Comment:
Maybe the default driver is called at some point. Because we didn't set it to Poltergeist, maybe Rack is called? You could try adding `Capybara.default_driver = :poltergeist` in `env.rb`.